plugins/qcheck-stm/doc/example_ghost.mli

```diff
@@ -0,0 +1,16 @@
+type 'a t
+(*@ model size : int
+    mutable model contents : 'a list *)
+
+val make : int -> 'a -> 'a t
+(*@ t = make i a
+    checks i >= 0
+    ensures t.size = i
+    ensures t.contents = List.init i (fun j -> a) *)
+
+(* $MDX part-begin=fun-decl *)
+
+val incompatible_type : char -> string t -> bool
+(*@ b = incompatible_type c t *)
```

Comment:
The example is called `example_ghost.mli` but this is not what it illustrates, so I’m a bit confused here. It was overwritten with the content of example_incompatible_type, it seems.